esx_service/vmdk_ops_test.py

```diff
@@ -685,7 +689,38 @@ class VmdkTenantTestCase(unittest.TestCase):
     datastore_name = None
     datastore_path = None
     
-
+    def create_fallback_tenant_and_privileges(self):
```

Comment:
As mentioned in above comment, we should check whether fallbacks need to be created even if table already exists. This way we don't need to do this in unit tests.